Metaverse/Source/Metaverse/MetaverseCharacter.cpp

```diff
@@ -218,3 +219,13 @@ void AMetaverseCharacter::HideQR() {
         Cast<AMetaverseHUD>(PlayerController->GetHUD());
     MetaverseHUD->HideQR();
 }
+
+UDynamicContractObject *
+AMetaverseCharacter::CreateDynamicContract(FString contractaddress,
+                                           FString abijson, bool &success,
```

Comment:
the abijson param is the ABI string? Does UE's builtin set of blueprints contain functions that can help with it (e.g. to load ABI from a file or URL)?